doc/source/getting_started.rst

```diff
@@ -14,37 +14,8 @@ Calour mostly handles **Experiment** data. An Experiment is made of **Samples**,
 
 Calour stores an Experiment as a synchronized set of: per-Sample metadata table (i.e. age, material, name, etc.), per-Feature metadata table (i.e. taxonomy, etc.) and a (sparse or dense) data matrix where position (i,j) stores the number of times (or frequency) feature i was observed in sample j.
 
-Calour contains severl functions for loading (or generating) such an Experiment. Additionally, Calour contains functions for filtering/reordering and statistical analysis of the Samples and Features. Finally, Calour can plot interactive heatmaps for exploring the Experiment and interfacing external databases (see **here** for example).
+Calour contains severl functions for loading (or generating) such an Experiment. Additionally, Calour contains functions for filtering/reordering and statistical analysis of the Samples and Features. Finally, Calour can plot interactive heatmaps for exploring the Experiment and interfacing external databases.
 
 Heatmap GUI
 -----------
 Calour can display the interactive heatmap either as a stand alone Qt5 window, or as an integrated jupyter notebook figure. The choice of the GUI is made in the `Experiment.plot()` function using the `gui='qt5'` or `gui='jupyter'` options. Additionaly, heatmaps can be generated as a non-interactive figure using the `gui='cli'` option.
-
-Running Calour
-==============
-Since calour is a python module, there is no need to run it. Instead, Calour is usually imported into the jupyter notebook you are using for analysis.
-
-For a full GUI using Calour, install `EZCalour <https://github.com/amnona/EZCalour>`_.
-
-
-Starting a Calour analysis notebook
------------------------------------
-If you installed Calour in a conda environment (usually the environment is named calour), first activate the environment:
-
-```source activate calour``` (in max/linux)
-
-or
-
-```activate calour``` (in windows)
-
-In order to run the jupyter notebook server, change directory to the directory where your analysis notebook is located and then type:
-
-```jupyter notebook```
-
-and within the notebook, just import the Calour module using:
-
-```import calour as ca```
-
-For a generic Calour microbiome analysis notebook that contains the standard analysis workflow, you can download the `simple_microbiome_analysis.ipynb <https://raw.githubusercontent.com/biocore/calour/master/notebooks/demo.ipynb>`_.
```

Comment:
Did it move to a different file? or just deleted? If the latter, i think we should keep something like this (from my experience with microbiome analysis users, some will get stuck on how to run calour and use it for the first time... Maybe should put in a different file/section...?

Reply:
deleted. I documented that your tutorial notebooks is available to download to run. if the case, could you put this notebook as the first tutorial notebook in the tutorial section later? BTW, I think you should re-order the notebooks in that section later - some notebooks should probably be put before others.